packages/graphql/lib/src/core/query_options.dart

```diff
@@ -118,12 +124,106 @@ class MutationOptions extends BaseOptions {
     FetchPolicy fetchPolicy,
     ErrorPolicy errorPolicy,
     Map<String, dynamic> context,
```

Comment:
```suggestion
    Map<String, dynamic> context,
    Object optimisticResult,
```

Reply:
I think this needs to be more thought out implementation and should be a separate issue. It is not as easy as moving existing APIs from one package to another.